### ISSUE TITLE
pub/sub: should publish a message and receive it again

### ISSUE THREAD

**Issue by raboof** [akka/alpakka#2446]
https://travis-ci.com/github/akka/alpakka/jobs/398247567
```
--> [docs.scaladsl.IntegrationSpec: pub/sub should publish a message and receive it again] Start of log messages of test that [Failed(org.scalatest.exceptions.TestFailedException: java.lang.RuntimeException: Unexpected pull response. Code: [500 Internal Server Error]. Entity: [{"error":{"code":500,"message":"Panic! This is a bug!","status":"INTERNAL"}}] was not an instance of java.util.concurrent.TimeoutException, but an instance of java.lang.RuntimeException)]
```

Seems like this is an error message that can be emitted by grpc-java - is grpc-java used in our project or in the emulator we are testing against?

**Comment by raboof** [comment]
possible duplicate of #2448

**Comment by raboof** [comment]
Hmm, again (https://travis-ci.com/github/akka/alpakka/jobs/402426335) though there the emulator is pinned to 313.0.1 :/

**Comment by raboof** [comment]
https://travis-ci.com/github/akka/alpakka/jobs/403083481

**Comment by ennru** [comment]
To pin the emulator fixed the gRPC client but not the HTTP client.
This makes me think the emulator has a problem `{"error":{"code":500,"message":"Panic! This is a bug!","status":"INTERNAL"}}`.

**Comment by raboof** [comment]
pub/sub succeeded after #2454 was merged (e.g. https://travis-ci.com/github/akka/alpakka/builds/192606797), let's hope that was it for now...

(of course we'd eventually want to test against a recent version of the emulator, but that is another issue ;) )
